packages/mobile/src/fees/selectors.ts

```diff
@@ -29,19 +29,21 @@ export const getReclaimEscrowFeeEstimateDollars = feeEstimateDollarsSelectorFact
   getReclaimEscrowFeeEstimateInWei
 )
 
-export const getFeeEstimateDollars = (state: RootState, feeType: FeeType | null) => {
-  if (feeType === null) {
-    return undefined
-  }
+export const getFeeEstimateDollars = (feeType: FeeType | null) => {
```

Comment:
while youre changing this anyway want to make it into a named function? 

Thats the guideline we have right? standalone functions named. class methods as arrows?